HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -146,7 +146,7 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
 
     // Separate SBATCH options from commands
     job.commands = editor.getValue();
-    var other = editor.getValue().split("\n");
+    var other = editor.getValue().split(/\r?\n/);
```

Comment:
I think so, after I submitted the job, I did `cat -A test.slurm` on crane and the new lines were all dollar signs so I assumed that this meant "\n"